src/styles/main.scss

```diff
@@ -1,3 +1,386 @@
-body {
-  margin: 0;
+.container {
+  background-color: $main-color;
+
+  position: relative;
+
+  img {
+    width: 100%;
+
+    display: block;
+  }
+
+  .main {
+    padding: 60px 20px 20px;
+
+    img {
+      transition: transform 0.3s ease;
+
+      &:hover {
+        transform: scale(1.05);
+      }
+    }
+
+    h1,
```

Comment:
Modifier Issues:

Modifiers like h1, h2, and legend are not in the BEM format. Instead of using these globally within .main, define specific classes or modifiers for state or style variations.